### PR TITLE
Add accesskey 's' to Run button

### PR DESCRIPTION
**Pull request by jhsoby**
Add accesskey to Run button for easier keyboard navigation.

@lucaswerkmeister I know this is not the right repo to do this in, but I have no idea how to submit a patch on the main repo where QuickStatements is hosted, but I noticed that you do, so I'm hoping you could please submit this there as well. :-)

### CODE REVIEW

**Comment by lucaswerkmeister** [comment]
Well, not really, I just asked him on Twitter to pull the commits :) (I did also use `git send-email`, but I think he ignored that :P ) but if you don’t have a Twitter account, I can point him here.

File: public_html/index.html [hunk 1]
```diff
@@ -188,7 +188,7 @@ <h1>Batches</h1>
 
 <div class='row' id='logged_in_actions'>
 <span id='run_buttons'>
-<button class='btn btn-lg btn-primary' id='run' tt='run'></button>
+<button class='btn btn-lg btn-primary' id='run' tt='run' accesskey='s'></button>
```
Comment (lucaswerkmeister) [inline public_html/index.html]: Neat, I didn’t know about `accesskey`!

Comment (jhsoby) [reply public_html/index.html]: Really? It's massively useful, especially on Wikipedia/Wikidata, plenty of things have accesskeys. R for Recent changes, L for watchlist, E for edit, T for talk page, C for content page, H for history, Z for main page, S for save, etc etc etc. Makes navigation much more efficient!

**Comment by jhsoby** [comment]
I sent him a tweet, so hopefully he'll fix it. Thanks. :-)

**Pull request closed without merging**